backend/README.md

```diff
@@ -19,7 +19,11 @@ To get a local copy up and running follow these simple example steps.
 npm install npm@latest -g
 ```
 
-- [MongoDB](https://docs.mongodb.com/manual/installation/)
+- [MongoDB](https://docs.mongodb.com/manual/installation/), make sure to copy .env.example to .env before start the backend server
```

Comment:
shouldn't this be `renamed` ?

Reply:
make sure to rename
Something like this

Reply:
yes, it can be renamed, but I'd prefer if we copy the `.env.example` because if we change something in the `.env` we will have a backup.

in the .env.example we will add more keys with example data, Does this make sense?
however, if you want I can put rename...

Reply:
Yeah, it does.